### ISSUE TITLE
On mac, option+left/right arrow to jump through words only works for english

### ISSUE THREAD

**Issue by NagisaVon** [notable/notable#1629]
- **OS Version**: darwin 12.4.0
- **Notable Version**: v1.9.0-beta.10

### Current behavior

On mac, option+left/right arrow to jump through words only works for english

### Expected behavior

That action should able to work with language other than english. 

here is a sentence for test... 

城市的敌人正在乌云上聚集
if you jump over word for this, the cursor should be land in-between like this:
城市 的 敌人 正在 乌云 上 聚集

**Comment by fabiospampinato** [comment]
> 城市 的 敌人 正在 乌云 上 聚集

To be clear each pair of characters here is a word, right?

The version of the editor used in beta10 was a little out of date, it should be much more up to date in beta11, that will probably fix it.

**Comment by NagisaVon** [comment]
> 

yes those pairs are words, I will check if it works when beta11 is out, thanks for the work!
